### PR TITLE
HOWTO: Prerelease and stable version

### PR DESCRIPTION
**Pull request by mtojek**
This PR introduces a short HOWTO paragraph about prereleases in Package Storage v2.

### CODE REVIEW

File: docs/howto/use_package_storage_v2.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ We recommend to use proper versioning instead and follow these rules:
...
+A flat storage structure has implications for exposing revisions in the Fleet Integrations UI.
+
+The Package Storage v1 allowed for pinning package revisions to particular stages and let package owners decide on when to promote the package.
+We received critical feedback about this approach as it required package owners to take a manual action to promote package revisions.
+The Package Storage v2 assumes that every published revision is exposed to Fleet users, and package owners can release their packages
+as snapshots or technical previews using semantic versioning (prerelease tags).
+
+Notice: if you prefer Fleet users not to pick up and install prereleases, it's recommended to be cautious with exposing new features
+until [kibana#122973](https://github.com/elastic/kibana/issues/122973) is implemented. With this feature, users can decide
+whether they can pick up prerelease package revisions.
```
Comment (jsoriano) [inline docs/howto/use_package_storage_v2.md:30]: Maybe we can also recommend to focus more on local/internal development before publishing prerelease packages till https://github.com/elastic/kibana/issues/122973 is implemented.

**Review by jsoriano — APPROVED** [review]

File: docs/howto/use_package_storage_v2.md (hunk 1 above)
Comment (mtojek) [reply docs/howto/use_package_storage_v2.md]: I added a sentence about the local dev environment.

**Review by jsoriano — APPROVED** [review]

**Comment by mtojek** [comment]
@v1v Another bug:

```
[2022-09-05T14:05:58.608Z] BucketNotFoundException: 404 gs://fleet-ci-temp-internal bucket does not exist.
```

**Comment by v1v** [comment]
I'll review it now, I recall It was created, but maybe i'm confused with a different name.

**Pull request merged**